packages/subgraph/schema.graphql

```diff
@@ -462,152 +452,356 @@ type StandardCallback @entity {
   magicNumber: Bytes!
 }
 
-# DAO
+#############################
+########### DAO #############
+#############################
 
+"""
+The Aragon DAOs deployed using the `DAOFactory` SC `createDAO` function and registered in the `DAORegistry` SC. From this entity you are able to query
+any relevant information related to your DAO.
+"""
 type Dao @entity {
-  id: ID! # use address as id
+  " The address of the deployed DAO contract registered in the `DAORegistry` SC. "
+  id: ID!
+
+  " ENS DAO subdomain, if added while creating the DAO. "
   subdomain: String
+
+  " The address of the creator who called (msg.sender) `createDao` in the `DAOFactory` SC. "
   creator: Bytes!
+
+  " The DAO uri used with [EIP-4824](https://eips.ethereum.org/EIPS/eip-4824), if added while creating the DAO. "
   daoURI: String
+
+  " Additional information related to the created DAO, if added while creating the DAO. "
   metadata: String
+
+  " Timestamp for the block where the DAO was created. "
   createdAt: BigInt!
+
+  " Tokens held by the DAO. "
   token: ERC20Contract
+
+  " Actions executed by the DAO `execute` function (refer to `Objects/Action` in this documentation site for more information). "
   actions: [Action!]! @derivedFrom(field: "dao")
+
+  " Token transfers associated with the DAO (refer to `Interfaces/TokenTransfer` in this documentation site for more information). "
   transfers: [TokenTransfer!]! @derivedFrom(field: "dao")
+
+  " DAO balances for all kinds of tokens (refer to `Interfaces/TokenBalance` in this documentation site for more information). "
   balances: [TokenBalance!]! @derivedFrom(field: "dao")
+
+  " The permissions given to a DAO to govern its actions (refer to `Objects/Permission` in this documentation site for more information). "
   permissions: [Permission!]! @derivedFrom(field: "dao")
+
+  " The proposals to execute actions that a DAO has had. "
```

Comment:
needs a better writing IMO, perhaps something like `A list of proposals for actions to be executed by the DAO, including both open and executed proposals.`